packages/web-runtime/src/layouts/Application.vue

```diff
@@ -208,14 +199,6 @@ export default defineComponent({
         })
       })
 
-      // Get extensions manually added into config
-      this.configuration.applications.forEach((application) => {
-        list.push({
-          ...application,
-          type: 'link'
-        })
-      })
-
```

Comment:
Not 100% sure, but I guess this was old (oC10) code.

Reply:
Looks superfishy at least